src_files.yml

```diff
@@ -50,6 +50,6 @@ ibex_regfile_fpga:
   incdirs: [
   ]
   files: [
-    rtl/ibex_register_file_ff.sv,
+    rtl/ibex_register_file_fpga.sv,
```

Comment:
Same comment as above, keep using the FF-based Regfile by default.

Reply:
Done. Thank you!